__fixtures__/test-project/package.json

```diff
@@ -21,8 +21,5 @@
   "prisma": {
     "seed": "yarn rw exec seed"
   },
-  "packageManager": "yarn@3.2.2",
-  "resolutions": {
-    "jest": "28.1.3"
-  }
+  "packageManager": "yarn@3.2.2"
```

Comment:
We had to add manual resolutions here to get jest 28 working. An unfortunately generating a new test project removes our manual modification. Can you please add it back in?

This is just temporary. As soon as we release v3 of Redwood we won't need these resolutions anymore.

Reply:
Sure thing (again, implicit knowledge). I simply did what the CI pipeline told me and blindly committed it :see_no_evil: 

Dropped that commit now.